cmd/juju/cloud/add.go

```diff
@@ -216,10 +220,15 @@ func (c *AddCloudCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.StringVar(&c.CloudFile, "f", "", "The path to a cloud definition file")
 	f.StringVar(&c.CloudFile, "file", "", "The path to a cloud definition file")
 	f.StringVar(&c.credentialName, "credential", "", "Credential to use for new cloud")
+	f.StringVar(&c.targetController, "target-controller", "", "The name of the controller managed by JAAS")
```

Comment:
"The name of a JAAS managed controller to add a cloud to".

Or something like that.